tests/mssql_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build integration
 // +build integration
```

Comment:
do we need them both?

Reply:
I've left it updated to the newer `go:build` directive